cscs-checks/cuda/multi_gpu.py

```diff
@@ -65,7 +66,8 @@ def __init__(self, **kwargs):
         self.tags = {'production'}
 
     def setup(self, partition, environ, **job_opts):
-        if self.current_system.name == 'kesch' and environ.name == 'PrgEnv-gnu':
+        if self.current_system.name == 'kesch' \
+                and environ.name == 'PrgEnv-gnu':
```

Comment:
Style comment: (a) we use parentheses for line continuation instead of `\`, (b) we split long lines after binary operators:

```python
        if (self.current_system.name == 'kesch' and
            environ.name == 'PrgEnv-gnu'):
```